README.md

```diff
@@ -1 +1 @@
-# Voyage Lanka [![Netlify Status](https://api.netlify.com/api/v1/badges/2ca61c7b-b464-4236-b6ea-d0e9d212af15/deploy-status?branch=main)](https://app.netlify.com/sites/voyagelanka/deploys)
+# Smart-Hotel-Booking
```

Comment:
Need to preserve previous changes n this file. Therefore, do not change this file.